control/pzmap.py

```diff
@@ -40,11 +40,42 @@
 #
 # $Id:pzmap.py 819 2009-05-29 21:28:07Z murray $
 
-from numpy import real, imag
-from .lti import LTI
+from numpy import real, imag, linspace, pi, exp, cos, sin, sqrt
+from .lti import LTI, isdtime, isctime
 
 __all__ = ['pzmap']
 
+def zgrid(plt):
```

Comment:
The zgrid function needs a docstring + some comments in the code to describe what is going on.

Reply:
Ok! Do you have any example I could be based on?

Reply:
Take a look at [nichols.py](https://github.com/python-control/python-control/blob/master/control/nichols.py) and, in particular, the `nichols_grid()` function.